core/templates/components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component.spec.ts

```diff
@@ -81,6 +81,15 @@ describe('CkEditor copy toolbar', () => {
     expect(blurSpy).toHaveBeenCalled();
   });
 
+  it('should revert cursor to normal when modal is closed', () => {
+    component.toggleToolActive();
+    fixture.detectChanges();
+    expect(document.body.style.cursor).toBe('copy');
+    component.ngOnDestroy();
+    fixture.detectChanges();
+    expect(document.body.style.cursor).not.toBe('copy');
```

Comment:
Test the expected behavior i.e, we want the cursor to be set to `''`, right?

Also, add a new line between each test blocks (i.e, setup, baseline_verification, action & final_verification) to present the block visually:

```
component.toggleToolActive();
fixture.detectChanges();

expect(document.body.style.cursor).toBe('copy');

component.ngOnDestroy();
fixture.detectChanges();

expect(document.body.style.cursor).not.toBe('');
```

Reply:
@DubeySandeep I'm not sure how the test will interpret that - that's why I made the test to check if the cursor is not in copy mode or not.

Reply:
I have added the new lines - do check it out.